### PR TITLE
[runtime] fix official docker image check

### PR DESCRIPTION
**Pull request by Binyang2014**

### CODE REVIEW

File: src/kube-runtime/src/init.d/image_checker.py [hunk 1]
```diff
@@ -37,6 +37,13 @@ def _is_docker_hub_uri(uri):
     return False
 
 
+def _get_docker_repository_name(image_name):
+    pathes = image_name.split("/")
```
Comment (abuccts) [inline src/kube-runtime/src/init.d/image_checker.py]: paths

File: src/kube-runtime/test/test_image_checker.py [hunk 2]
```diff
@@ -94,6 +94,21 @@ def test_docker_hub_uri(self):
         for uri in invalid_docker_hub_uris:
             self.assertFalse(image_checker._is_docker_hub_uri(uri))
 
+    def test_get_docker_repository_name(self):
+        official_image_names = [
+            "ubuntu", "python:latest", "golang:1.12.6-alpine"
+        ]
+        third_party_image_name = [
```
Comment (abuccts) [inline src/kube-runtime/test/test_image_checker.py]: names

File: src/kube-runtime/src/init.d/image_checker.py (hunk 1 above)
Comment (Binyang2014) [reply src/kube-runtime/src/init.d/image_checker.py]: fixed

File: src/kube-runtime/test/test_image_checker.py (hunk 2 above)
Comment (Binyang2014) [reply src/kube-runtime/test/test_image_checker.py]: fixed

**Pull request merged**